pages/project-version/downloads/content/index.js

```diff
@@ -2,10 +2,10 @@ module.exports = {
   title: 'Downloads',
   licence: {
     inactive: {
-      heading: 'Draft licence',
-      link: 'Draft licence: {{title}} (.pdf)',
+      heading: 'Licence preview',
+      link: 'Licence preview: {{title}} (.pdf)',
       hint: 'Download a preview of how the licence will appear if granted. This doesn\'t include any restrictions the Home Office may add.',
-      back: 'Back to draft licence'
+      back: 'Back to licence preview'
```

Comment:
Is this particular change specced? It seems like the thing they'd be going back to at this point still _is_ the draft licence since it refers to the editing UI and not the PDF download.

Reply:
It is not, thought it'd just been missed as we were changing draft licence -> licence preview. I'll revert it.